gax-java/gax-httpjson/src/test/java/com/google/api/gax/httpjson/RetryingTest.java

```diff
@@ -200,6 +213,9 @@ void retryMaxAttemptsExceeded() {
         HttpJsonCallableFactory.createUnaryCallable(
             callInt, callSettings, httpJsonCallSettings, clientContext);
     assertThrows(ApiException.class, () -> callable.call(initialRequest));
+    assertThat(tracerFactory.getTracerAttempts().get()).isEqualTo(2);
+    assertThat(tracerFactory.getTracerAttemptsFailed().get()).isEqualTo(1);
```

Comment:
nit: Perhaps we can add TracerOperationFailed == true check?

Reply:
Done